godtools/GodToolsConstants.swift

```diff
@@ -0,0 +1,14 @@
+//
```

Comment:
Sure it could be a struct. I'll change that.

Do we need to prefix all the files with GT? I know that is common practice in older iOS apps, but is it still necessary? I find it clutters the filenames and makes them harder to read.

For the ones that do need prefixed, I do agree that GT is better than GodTools, so I'll change that for this file.